SonarQube.TeamBuild.PreProcessor/AnalysisConfigGenerator.cs

```diff
@@ -53,13 +53,19 @@ public static AnalysisConfig GenerateFile(ProcessedArgs args, TeamBuildSettings
                 AddSetting(config.ServerSettings, property.Key, property.Value);
             }
 
-            // Add command line and file arguments
+            // Add command line arguments
             config.LocalSettings = new AnalysisProperties();
-            foreach (var property in args.GetAllProperties())
+            foreach (var property in args.LocalProperties.GetAllProperties())
```

Comment:
We should only copy non-sensitive command line properties into the config file.
I'll rename LocalProperties to CmdLineProperties separately.